sympy/combinatorics/tests/test_pc_groups.py

```diff
@@ -79,12 +79,27 @@ def test_induced_pcgs():
     gens = [G[0], G[1]]
     ipcgs = collector.induced_pcgs(gens)
     order = [gen.order() for gen in ipcgs]
-    assert order == [3, 3]
+    assert order == [3, 3, 3, 3]
 
     G = SymmetricGroup(20).sylow_subgroup(2)
     PcGroup = G.polycyclic_group()
     collector = PcGroup.collector
     gens = [G[0], G[1], G[2], G[3]]
     ipcgs = collector.induced_pcgs(gens)
     order = [gen.order() for gen in ipcgs]
-    assert order ==[2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2]
+    assert order == [2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2]
+
+def test_issue_18804():
+    G = DihedralGroup(10)
```

Comment:
There is an existing function to test presentation, this can go to that.

Reply:
can you tell which one I am not able to find it.
or something like this:
```
>>> G = DihedralGroup(10)
>>> a = G.polycyclic_group()
>>> a.is_group
True
```

Reply:
I guess, the first function is to test pc_presentation which has a list containing multiple groups you can append this as well.

Reply:
ohhk got it.